packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -160,25 +142,17 @@ export function generateGCConfigs(
 	});
 
 	const throwOnInactiveLoad: boolean | undefined = createParams.gcOptions.throwOnInactiveLoad;
-
-	const throwOnTombstoneLoadConfig =
-		mc.config.getBoolean(throwOnTombstoneLoadOverrideKey) ??
-		createParams.gcOptions[gcDisableThrowOnTombstoneLoadOptionName] !== true;
 	const throwOnTombstoneLoad =
-		throwOnTombstoneLoadConfig && sweepAllowed && !createParams.isSummarizerClient;
-	const throwOnTombstoneUsage =
-		mc.config.getBoolean(throwOnTombstoneUsageKey) === true &&
-		sweepAllowed &&
+		mc.config.getBoolean(throwOnTombstoneLoadOverrideKey) !== false &&
```

Comment:
Typical logic for an override is:  If it's set, use the value true or false.  If it's unset, keep checking the other config sources.

As it is, you can only _disable_ throwing if sweep is enabled.  You cn'at _enable_ throwing when Sweep is disabled.

Is that what you intend?  I don't see a strong need for this setting, but maybe just rename it to `disableThrowOnTombstoneLoadKey` and check for `true`, to be clearer.

Reply:
Good point. With this PR, the intent is that sweep and tombstone are an atomic unit. However, we provide an escape hatch to disable tombstone only in case it causes too much disruption. So, I changed it to `disableThrowOnTombstoneLoadKey`